app/interactors/tasks/update.rb

```diff
@@ -3,5 +3,15 @@ class Update
     include Interactor::Organizer
 
     organize Tasks::Save
+
+    after do
+      task = context.task
+      project = task.project
+      users = project.users
```

Comment:
You can delegate task, and don't needed others
```suggestion
```
```suggestion
      task = context.task
      project = task.project
      users = project.users
```